tests/cross/src/test/scala/tests/pc/CompletionWorkspaceSuite.scala

```diff
@@ -610,4 +610,35 @@ class CompletionWorkspaceSuite extends BaseCompletionSuite {
        |""".stripMargin,
     filter = _ == "Implicits - scala.concurrent.ExecutionContext"
   )
+
+  checkEdit(
+    // it's needed to rework TextEdits.applyEdits
+    "specify-owner".tag(IgnoreScala2).ignore,
```

Comment:
```suggestion
    "specify-owner".ignore,
```
should it not work the same for Scala 2?

Same for the one below